hammers/scripts/clean_old_aggregates.py

```diff
@@ -20,15 +24,25 @@
 
 parser = base_parser(
     'Clean old Nova aggregates tied to expired Blazar leases.')
+mysqlargs = MySqlArgs({
+    'user': 'root',
+    'password': '',
+    'host': 'localhost',
+    'port': '3306',
+    })
+mysqlargs.inject(parser)
 args = parser.parse_args(sys.argv[1:])
 auth = osapi.Auth.from_env_or_args(args=args)
+mysqlargs.extract(args)
+conn = mysqlargs.connect()
 
-# Get aggregates and leases
+#Get aggregates, leases, host_allocations
```

Comment:
I'm not even sure this comment needs to exist, we could remove it. The code explains what it is doing well enough in my view